### PR TITLE
lock "history" package version to 5.0.1

### PR DESCRIPTION
**Pull request by alexrybch**
Signed-off-by: Alex Rybchenko <arybchenko@box.com>

## Hey, I just made a Pull Request!

Recent changes in "history" package are incompatible with "react-router-dom@6.0.0-beta.0" (`State` type removed https://github.com/remix-run/history/compare/v5.0.1...v5.0.2#diff-1caf4c08131a3efeb1e49c0ce896aa4a488a13b1b743613a99f4a5c5aff91d8fL57)

and e2e now fails:
https://github.com/backstage/backstage/runs/4077499724?check_suite_focus=true

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by freben** [comment]
Hi! Thanks. You'll need a changeset with one `patch` line per package changed, so that they will get published. I also expected to see yarn.lock` changes as part of the PR right?

**Comment by alexrybch** [comment]
> Hi! Thanks. You'll need a changeset with one `patch` line per package changed, so that they will get published. I also expected to see yarn.lock` changes as part of the PR right?

yeah, added lock file.
Sorry for missing changeset every time, will try to not forget to add it at once next time :)

File: .changeset/metal-apes-reply.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'example-app': patch
```
Comment (freben) [inline .changeset/metal-apes-reply.md:2]: ah now i have to be annoying - the non-released packages should not have a changeset line :) so remove this one, and then I think it's good to go.

**Comment by alexrybch** [comment]
ok, they put it back :) 
https://github.com/remix-run/history/compare/v5.0.3...v5.1.0

closing this pull request

**Comment by freben** [comment]
Nice! Thanks for being vigilant with this and following through.

**Pull request closed without merging**